telos-frontend/src/components/calendar/DashboardCalendar.jsx

```diff
@@ -50,14 +50,14 @@ const DashboardCalendar = () => {
   ]);
 
   return (
-    <div>
+    <>
```

Comment:
Not an issue, just curious, but what does using this instead of div do?

Reply:
After googling, it seems like a type that's not defined, basically used to wrap everything. Not sure why a div isn't used though.

Reply:
The div was only as big as it needed to be to fit its contents - the two options were to either use an empty fragment or to style the div to have height of 100%, either one works